### ISSUE TITLE
How to exclude items from item log

### ISSUE THREAD

**Issue by Deathknight55** [kolton/d2bot-with-kolbot#3111]
I am keeping flawless\perfect skulls to use for rolling but I really don't want to see it in the log every time one is kept.  Can anyone tell me how I would exclude that from the logs?  I already tried setting this line in my character config to false:
Config.ShowHighGems = false; // show/stop high gems (flawless, perfect) to appear in the item log

**Comment by DarkHorseDre** [comment]
The Config setting you quoted is incorrect and does not exist. download/use the original config file and settings

**Comment by Deathknight55** [comment]
> 
> 
> The Config setting you quoted is incorrect and does not exist. download/use the original config file and settings

Apparently in the newest one it has been changed to this:
Config.LogHighGems = false; // Log high gems (flawless, perfect) on item viewer

**Comment by DarkHorseDre** [comment]
Yup, always update your files or check the originals (you can click the "code" tab above and see the code without downloading it)

Please close this issue if your issue has been resolved
